libs/internal/src/serialization/json_flag.cpp

```diff
@@ -19,7 +19,7 @@ tl::expected<std::optional<data_model::Flag::Rollout>, JsonError> tag_invoke(
     REQUIRE_OBJECT(json_value);
     auto const& obj = json_value.as_object();
 
-    data_model::Flag::Rollout rollout;
+    data_model::Flag::Rollout rollout{};
```

Comment:
Because we deserialize all fields, nothing should be uninitialized garbage... but.. if we added a new field and tried to access it somewhere, while forgetting to deserialize it, then it would be. Better safe than sorry.